5.5/apache/docker-php-ext-install

```diff
@@ -4,16 +4,38 @@ set -e
 cd /usr/src/php/ext
 
 usage() {
-	echo "usage: $0 ext-name [ext-name ...]"
+	echo "usage: $0 [-jN] ext-name [ext-name ...]"
 	echo "   ie: $0 gd mysqli"
 	echo "       $0 pdo pdo_mysql"
+	echo "       $0 -j5 gd mbstring mysqli pdo pdo_mysql shmop"
 	echo
 	echo 'if custom ./configure arguments are necessary, see docker-php-ext-configure'
 	echo
 	echo 'Possible values for ext-name:'
 	echo $(find /usr/src/php/ext -mindepth 2 -maxdepth 2 -type f -name 'config.m4' | cut -d/ -f6 | sort)
 }
 
+opts="$(getopt -o 'h?j:' --long 'help,jobs:' -- "$@" || { usage >&2 && false; })"
+eval set -- "$opts"
+
+j=1
```

Comment:
I wonder if this should default to `$(nproc)`.

Reply:
https://github.com/docker-library/drupal/pull/154#discussion_r297898483:

>It defaults to `make`'s default (we don't explicitly `-j1` anywhere), mostly to be considerate of `php` users' machines -- for the official images though, we want to make full use of our build systems as much as possible (especially chasing the dragon of potentially faster builds), which is why we typically use `-j "$(nproc)"` in ours.